launch/microstrain.launch

```diff
@@ -3,7 +3,7 @@
   <!-- Standalone example launch file for 3DM-GX5-25 -->
 
   <!-- Declare arguments with default values -->
-  <arg name="port" default="/dev/microstrain" />
+  <arg name="port" default="/dev/ttyACM0" />
```

Comment:
Is the udev rule not working for you?

Reply:
Done, forgot to put it back. I actually launch the driver from a higher level launch file and rarely use this one aside from testing and forgot to put it back apparently. :)